src/content/docs/tutorial-create-alerts/create-nrql-alerts.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: "Create a NRQL alert"
+metaDescription: "How to create a NRQL alert with New Relic"
+---
+
+import alertsNRQL from 'images/alerts_screenshot-crop_NRQL-query.webp'
+
+import alertsNewCreate from 'images/alerts_screenshot-crop_create-new-query.webp'
+
+While using charts is a great way to create alert conditions from premade queries, sometimes you need to create alert conditions from scratch. The process for creating a new alert from a NRQL query is almost the exact same as creating it from a chart: the key difference is that you write the NRQL query manually instead of using a pre-generated one. 
+
+<img
+    title="NRQL alert"
+    alt="A screenshot displaying an alert condition created with a simple NRQL query"
+    src={alertsNRQL}
+/>
+
+<Callout variant="tip">
+Creating alert conditions from NRQL queries requires a basic understand of NRQL, New Relic's Query Language. See our [NRQL docs](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) to learn more about NRQL, including [how to use it](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works/) and an [introductory tutorial](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial/).
+    </Callout>
+
+<Steps>
+    <Step>
+## Create an alert 
+
+<img
+    title="New alert policy"
+    alt="A screenshot displaying where to create new alert conditions in New Relic"
+    src={alertsNewCreate}
+/>
+
+1. From **one.newrelic.com**, select [**Alerts & AI**](https://one.newrelic.com/alerts-ai), the [**Alert conditions (policies)**](https://one.newrelic.com/alerts-ai/alerts-classic/policies).
+
+2. Select **New alert condition**.
+
+3. Write the NRQL query to create the condition. For example, if you wanted to monitor slow transactions, you can enter:
+```sql
+SELECT average(duration) from Transaction facet name
+```
+
+4. Select **Next**.
+    </Step>
+    <Step>
+## Adjust the signal behavior
+
+1. Set the data aggregation options. You will need to:
+    * Use the **Window duration** setting to determine how frequently we aggregate your data, such as groupings of every 5 minutes or every hour
```

Comment:
Similar to the chart alerts doc, these longer bullets would really benefit from bolded phrases to highlight the main idea fo reach bulle tand enable scanning.

Reply:
Fixed!